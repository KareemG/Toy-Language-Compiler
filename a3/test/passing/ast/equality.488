% Equality operation

{
    if 5 = 5 then
        write "PASS"
    else
        write "FAIL"
}