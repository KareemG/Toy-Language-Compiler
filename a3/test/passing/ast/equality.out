 { 
 if (5) = (5) then
 write "PASS" else
 write "FAIL"end
 }
